src/dialects/postgres/schema/tablecompiler.js

```diff
@@ -90,6 +90,11 @@ TableCompiler_PG.prototype.comment = function(comment) {
   this.pushQuery(`comment on table ${this.tableName()} is '${this.single.comment}'`);
 };
 
+TableCompiler_PG.prototype.check = function(logic) {
+  const name = this.formatter.wrap(`${this.tableNameRaw}_check`);
```

Comment:
I think this naming convention limits number of check constraints per table to be 1.